source/D2Lang/src/D2Unicode/D2UnicodeStr.cpp

```diff
@@ -24,6 +24,17 @@
 
 #include <D2Unicode.h>
 
+Unicode* __fastcall Unicode::strcpy(Unicode* dest, const Unicode* src) {
+  size_t i;
```

Comment:
Could you make it `size i = 0;` instead of `size i; i = 0;` ?

Reply:
Done.